audformat/core/utils.py

```diff
@@ -67,6 +71,42 @@ def concat(
         f1   0 days NaT    0.0    NaN
         f2   0 days NaT    1.0      a
         f3   0 days NaT    2.0      b
+        >>> obj1 = pd.Series(
+        ...     [0., 1.],
+        ...     index=filewise_index(['f1', 'f2']),
+        ...     name='float',
+        ... )
+        >>> obj2 = pd.DataFrame(
+        ...     {
+        ...         'float': [np.nan, 2.],
+        ...         'string': ['a', 'b'],
+        ...     },
+        ...     index=segmented_index(['f2', 'f3']),
```

Comment:
Maybe change this also to a filewise index to show that in this case it will also return a filewise index.

Reply:
Good idea. I changed it accordingly.